src/RL/representation/default/defaultstaterepresentation.jl

```diff
@@ -149,6 +158,19 @@ function featurize(sr::DefaultStateRepresentation{DefaultFeaturization,TS}; chos
             end
             if sr.chosenFeatures["constraint_type"][1]
                 features[sr.constraintTypeToId[typeof(cp_vertex.constraint)], i] = 1
+                if isa(cp_vertex.constraint, ViewConstraint)
+                    if isa(cp_vertex.constraint.child, IntVarViewMul)
+                        features[sr.constraintTypeToId[typeof(cp_vertex.constraint)] + 1, i] = cp_vertex.constraint.child.a
+                    elseif isa(cp_vertex.constraint.child, IntVarViewOffset)
+                        features[sr.constraintTypeToId[typeof(cp_vertex.constraint)] + 2, i] = cp_vertex.constraint.child.c
+                    elseif isa(cp_vertex.constraint.child, IntVarViewOpposite)
+                        features[sr.constraintTypeToId[typeof(cp_vertex.constraint)] + 1, i] = -1
+                    elseif isa(cp_vertex.constraint.child, BoolVarViewNot)
+                        features[sr.constraintTypeToId[typeof(cp_vertex.constraint)] + 3, i] = 1
+                    else
+                        error("WARNING: Unknwon VarViewType: please implement DefaultFeaturization for this type!")
+                    end
+                end
```

Comment:
If the problem we are trying to solve has `ViewConstraints`, in `constraint_type`, we encode them thanks to 4 new components, respectively: one-hot encoding of the fact the considered constraint is a `ViewConstraint`, multiplication coefficient if the `ViewConstraint` is of type `IntVarViewMul` or `IntVarViewOpposite`, offset coefficient if the if the `ViewConstraint` is of type `IntVarViewOffset`, and one-hot encoding of the fact the `ViewConstraint` is of type `BoolVarViewNot`.